app/enterprise/2.3.x/external-plugins.md

```diff
@@ -41,9 +41,9 @@ an hypothetical Python plugin server called `pypluginserver.py`):
 ```
 pluginserver_names = go,python
 
-pluginserver_go_socket = /usr/local/kong/go_pluginserver.socket
-pluginserver_go_start_cmd = go-pluginserver -kong-prefix /usr/local/kong/ -plugins-dir /usr/local/kong/go-plugins
-pluginserver_go_query_cmd = go-pluginserver -dump-all-plugins -plugins-dir /usr/local/kong/go-plugins
+pluginserver_go_socket = /usr/local/kong/go_pluginserver.sock
+pluginserver_go_start_cmd = go-pluginserver -kong-prefix /usr/local/kong/ -plugins-directory /usr/local/kong/go-plugins
+pluginserver_go_query_cmd = go-pluginserver -dump-all-plugins -plugins-directory /usr/local/kong/go-plugins
 
 pluginserver_python_socket = /usr/local/kong/pyton_pluginserver.socket
```

Comment:
@javierguerragiraldez should `pyton` be `python`? And should this one also be `sock` instead of `socket`?

/usr/local/kong/python_pluginserver.sock

Reply:
there's no "official" python pluginserver, so it doesn't really matter.  but yes, I think that would be more readable